Sources/Model/TransferMethod/HyperwalletTransferMethodQueryParam.swift

```diff
@@ -56,12 +58,33 @@ public class HyperwalletTransferMethodQueryParam: QueryParam {
         case descendantStatus = "-status"
     }
 
+    /// Representation of the transfer method type
+    public enum QueryType: String {
+        /// The Bank account type
+        case bankAccount = "BANK_ACCOUNT"
+        /// The Wire account type
+        case wireAccount = "WIRE_ACCOUNT"
+        /// The Bank Card type
+        case bankCard = "BANK_CARD"
+        /// The Paypal account type
+        case paypalAccount = "PAYPAL_ACCOUNT"
+        /// The Venmo account type
+        case venmoAccount = "VENMO_ACCOUNT"
+        /// The Prepaid card type
+        case prepaidCard = "PREPAID_CARD"
+        /// The Paper check type
+        case paperCheck = "PAPER_CHECK"
+    }
+
     override public func toQuery() -> [String: String] {
         var query = super.toQuery()
 
         if let status = status {
             query[QueryParam.status.rawValue] = status.rawValue
         }
+        if let type = type {
+            query[QueryParam.type.rawValue] = type.rawValue
+        }
```

Comment:
If you are already setting it here why do you need this logic in every other subclassed QueryParam class?